app/assets/javascripts/charts.js

```diff
@@ -56,13 +56,12 @@ var Charts = {
         var first_day = new Date(this.value)
         var last_day = new Date(first_day.getFullYear(), first_day.getMonth()+1, 0);
         var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
-        return '';
-        // if (months.includes(this.value.split(' ')[0]))
-          // return '<a href="/admin/accounts?commit=Filter&q[created_at_gteq_datetime]=' + first_day + '&q[created_at_lteq_datetime]=' + last_day + '" target="_blank">' +
-                    // this.value + '</a>';
-        // else
-          // return '<a href="/admin/accounts?commit=Filter&q[created_at_gteq_datetime]=' + this.value + '&q[created_at_lteq_datetime]=' + this.value + '" target="_blank">' +
-                    // this.value + '</a>';
+        if (months.includes(this.value.split(' ')[0]))
+          return '<a href="/admin/accounts?commit=Filter&q[created_at_gteq_datetime]=' + first_day + '&q[created_at_lteq_datetime]=' + last_day + '" target="_blank">' +
+                    this.value + '</a>';
+        else
+          return '<a href="/admin/accounts?commit=Filter&q[created_at_gteq_datetime]=' + this.value + '&q[created_at_lteq_datetime]=' + this.value + '" target="_blank">' +
+                    this.value + '</a>';
```

Comment:
The above code was commented out on the Main branch. This code has been done on OTWO-6652 providing the link to the account trend graph which will go to the account page.

So added this one with this commit only.